main.go

```diff
@@ -63,6 +64,7 @@ func init() {
 	kingpin.Flag("create-events", "If true, create an event in victims namespace after termination.").Default("true").BoolVar(&createEvent)
 	kingpin.Flag("debug", "Enable debug logging.").BoolVar(&debug)
 	kingpin.Flag("metrics-address", "Listening address for metrics handler").Default(":8080").StringVar(&metricsAddress)
+	kingpin.Flag("grace-period", "Grace period in seconds to terminate Pods").Default("30").Int64Var(&gracePeriod)
```

Comment:
Since we're importing `client-go` anyways we may be able to use a constant defined somewhere in there for that value. Just a thought 😃

Reply:
Yes, I thought about referencing the `30` in the Kubernetes package where the default grace period for the pod is defined. But I don't understand very well the organization of Kubernetes source code and I was't able to find it.

Reply:
That's understandable. I also can't fine it. Let's leave it like it is.